commit 5cd5746f701abd54da124cbcf3d2482a67364a1c
Author: Saeed Mahameed <saeedm@mellanox.com>
Date:   Wed Feb 22 17:20:12 2017 +0200

    net/mlx5e: Register/unregister vport representors on interface attach/detach
    
    [ Upstream commit 6f08a22c5fb2b9aefb8ecd8496758e7a677c1fde ]
    
    Currently vport representors are added only on driver load and removed on
    driver unload.  Apparently we forgot to handle them when we added the
    seamless reset flow feature.  This caused to leave the representors
    netdevs alive and active with open HW resources on pci shutdown and on
    error reset flows.
    
    To overcome this we move their handling to interface attach/detach, so
    they would be cleaned up on shutdown and recreated on reset flows.
    
    Fixes: 26e59d8077a3 ("net/mlx5e: Implement mlx5e interface attach/detach callbacks")
    Signed-off-by: Saeed Mahameed <saeedm@mellanox.com>
    Reviewed-by: Hadar Hen Zion <hadarh@mellanox.com>
    Reviewed-by: Roi Dayan <roid@mellanox.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1371_5cd5746f701abd54da124cbcf3d2482a67364a1c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1371_5cd5746f701abd54da124cbcf3d2482a67364a1c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
